Add CVS support

pnowakowski opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/heremaps/oss-review-toolkit/135)
<!-- Reviewable:end -->

mnonnenmacher commented:
Reviewed 8 of 8 files at r1, 5 of 5 files at r2, 2 of 2 files at r3.
Review status: all files reviewed at latest revision, 12 unresolved discussions, some commit checks failed.

---

*[analyzer/src/funTest/kotlin/GradleTest.kt, line 26 at r2](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JMbdp_38HgqYAiZTV:-L0JMbdp_38HgqYAiZTW:b-ufayxp) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/9fb84d1f26af5453201dff64569b33fd828349c6/analyzer/src/funTest/kotlin/GradleTest.kt#L26)):*
> ```Kotlin
> 
> import com.here.ort.util.normalizeVcsUrl
> import com.here.ort.downloader.WorkingDirectoryWithRevision
> ```

Put in alphabetical order.

---

*[analyzer/src/funTest/kotlin/GradleTest.kt, line 39 at r2](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JMf7fbGkZ_rZHtvAB:-L0JMf7fbGkZ_rZHtvAC:b-skw5js) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/9fb84d1f26af5453201dff64569b33fd828349c6/analyzer/src/funTest/kotlin/GradleTest.kt#L39)):*
> ```Kotlin
> class GradleTest : StringSpec() {
>     private val projectDir = File("src/funTest/assets/projects/synthetic/gradle")
>     private val vcsDir = VersionControlSystem.forDirectory(projectDir)!! as WorkingDirectoryWithRevision
> ```

Is this an unsafe cast?

---

*[downloader/src/funTest/kotlin/CvsTest.kt, line 52 at r2](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JM3rVAlG0QsMAZllh:-L0JM3rWLvCaGBfTMO9s:b-gh9dw6) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/9fb84d1f26af5453201dff64569b33fd828349c6/downloader/src/funTest/kotlin/CvsTest.kt#L52)):*
> ```Kotlin
> 
>     init {
>         "Detected Cvs version is not empty" {
> ```

In strings use uppercase CVS (also below).

---

*[downloader/src/funTest/kotlin/CvsTest.kt, line 76 at r2](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JMEFwUmS8Kfp6hcea:-L0JMEFxv6en9FrvMfyz:b-hq0k5c) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/9fb84d1f26af5453201dff64569b33fd828349c6/downloader/src/funTest/kotlin/CvsTest.kt#L76)):*
> ```Kotlin
>         }
> 
>         "Cvs checkouts whole CVS repository" {
> ```

Better say: "CVS can check out ..." here and below.

---

*[downloader/src/main/kotlin/VersionControlSystem.kt, line 180 at r1](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JGR9NGY9K9opnRBQg:-L0JGR9NGY9K9opnRBQh:b-r1zonp) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/c51376263402937f717199616bfeda4bb0040613/downloader/src/main/kotlin/VersionControlSystem.kt#L180)):*
> ```Kotlin
>      */
>     abstract fun download(vcsUrl: String, vcsRevision: String?, vcsPath: String?, version: String, targetDir: File)
>             : String?
> ```

The empty string is used as a revision in VcsInfo below for WorkingDirectoryWithFileRevisions. I think we should consistently use the empty string then and not allow this to return null.

---

*[downloader/src/main/kotlin/VersionControlSystem.kt, line 215 at r1](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JD0uOOW4PCGTnnUfp:-L0JD0uQj8c-_AprldUG:b-ebj54v) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/c51376263402937f717199616bfeda4bb0040613/downloader/src/main/kotlin/VersionControlSystem.kt#L215)):*
> workingFir
workingDir

---

*[downloader/src/main/kotlin/VersionControlSystem.kt, line 226 at r1](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JD6jUrJHpaFkBcTiq:-L0JD6jUrJHpaFkBcTir:b-ebj54v) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/c51376263402937f717199616bfeda4bb0040613/downloader/src/main/kotlin/VersionControlSystem.kt#L226)):*
> workingFir
workingDir

---

*[downloader/src/main/kotlin/vcs/Cvs.kt, line 36 at r2](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JHGnh3q1mtIHI4Ilh:-L0JHGniTOuExC76axWP:b6awt3b) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/9fb84d1f26af5453201dff64569b33fd828349c6/downloader/src/main/kotlin/vcs/Cvs.kt#L36)):*
> ```Kotlin
> object Cvs : VersionControlSystem<WorkingDirectoryWithFileRevisions>() {
>     override fun getVersion(): String {
>         val mercurialVersionRegex = Regex("Concurrent Versions System \\(CVS\\) (?<version>[\\d.]+).+")
> ```

Rename to cvsVersionRegex.

---

*[downloader/src/main/kotlin/vcs/Cvs.kt, line 56 at r2](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JIJSyftyciqD-UpJF:-L0JIJSyftyciqD-UpJG:b-2b60tu) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/9fb84d1f26af5453201dff64569b33fd828349c6/downloader/src/main/kotlin/vcs/Cvs.kt#L56)):*
> ```Kotlin
>                     val revisionPrefix = "Working revision:"
>                     return commandOutput.first { it.trim().startsWith(revisionPrefix) }
>                             .trim().removePrefix(revisionPrefix).trim()
> ```

You can save one trim:
`commandOutput.map { it.trim() }.first { it.startsWith(revisionPrefix) }.removePrefix(revisionPrefix).trim()`

---

*[downloader/src/main/kotlin/vcs/Cvs.kt, line 61 at r2](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JInw1JQsnmID4OjVJ:-L0JInw1JQsnmID4OjVK:b-zbuncg) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/9fb84d1f26af5453201dff64569b33fd828349c6/downloader/src/main/kotlin/vcs/Cvs.kt#L61)):*
> ```Kotlin
>                 private val cvsDir = File(workingDir, "CVS")
> 
>                 override fun isValid(): Boolean
> ```

Can we omit the return type here?

---

*[downloader/src/main/kotlin/vcs/Cvs.kt, line 64 at r2](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JIxyxuksEp3jPdRTh:-L0JIxyxuksEp3jPdRTi:b-zbuncg) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/9fb84d1f26af5453201dff64569b33fd828349c6/downloader/src/main/kotlin/vcs/Cvs.kt#L64)):*
> ```Kotlin
>                         = ProcessCapture(workingDir, "cvs", "status").exitValue() == 0
> 
>                 override fun getRemoteUrl(): String = File(cvsDir, "Root").readLines().first().substringAfter("/")
> ```

Can we omit the return type here?

---

*[downloader/src/main/kotlin/vcs/Cvs.kt, line 78 at r2](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JLHM1Htgh0T2ePjCs:-L0JLHM2l-ML_GaBWNeD:bqcpinv) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/9fb84d1f26af5453201dff64569b33fd828349c6/downloader/src/main/kotlin/vcs/Cvs.kt#L78)):*
> ```Kotlin
>                         path.parent
>                     }
>                     return File(parent, "CVS/Repository").readLines().first()
> ```

This provides much better performance for large files, because it does not read the full file to get only the first line:
`File(...).useLines { it.first() }`

You have this in several places, please replace all of them.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135)*
<!-- Sent from Reviewable.io -->

pnowakowski commented:
Review status: 4 of 13 files reviewed at latest revision, 12 unresolved discussions, some commit checks failed.

---

*[analyzer/src/funTest/kotlin/GradleTest.kt, line 26 at r2](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JMbdp_38HgqYAiZTV:-L0JQdZFEjITKtN1xLi-:b-896fix) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/9fb84d1f26af5453201dff64569b33fd828349c6/analyzer/src/funTest/kotlin/GradleTest.kt#L26)):*
<details><summary><i>Previously, mnonnenmacher (Martin Nonnenmacher) wrote…</i></summary><blockquote>

Put in alphabetical order.
</blockquote></details>

Done.

---

*[analyzer/src/funTest/kotlin/GradleTest.kt, line 39 at r2](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JMf7fbGkZ_rZHtvAB:-L0JRJTW07K_fw2HPdOO:bj824am) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/9fb84d1f26af5453201dff64569b33fd828349c6/analyzer/src/funTest/kotlin/GradleTest.kt#L39)):*
<details><summary><i>Previously, mnonnenmacher (Martin Nonnenmacher) wrote…</i></summary><blockquote>

Is this an unsafe cast?
</blockquote></details>

Well, it is, but here, in test we know exactly what kind of working directory there is, so I assumed it's ok

---

*[downloader/src/funTest/kotlin/CvsTest.kt, line 52 at r2](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JM3rVAlG0QsMAZllh:-L0JSQp6K4Y7vvPXEmUD:b-896fix) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/9fb84d1f26af5453201dff64569b33fd828349c6/downloader/src/funTest/kotlin/CvsTest.kt#L52)):*
<details><summary><i>Previously, mnonnenmacher (Martin Nonnenmacher) wrote…</i></summary><blockquote>

In strings use uppercase CVS (also below).
</blockquote></details>

Done.

---

*[downloader/src/funTest/kotlin/CvsTest.kt, line 76 at r2](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JMEFwUmS8Kfp6hcea:-L0JSbTQO5Bm8NzixgB2:b-896fix) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/9fb84d1f26af5453201dff64569b33fd828349c6/downloader/src/funTest/kotlin/CvsTest.kt#L76)):*
<details><summary><i>Previously, mnonnenmacher (Martin Nonnenmacher) wrote…</i></summary><blockquote>

Better say: "CVS can check out ..." here and below.
</blockquote></details>

Done.

---

*[downloader/src/main/kotlin/VersionControlSystem.kt, line 180 at r1](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JGR9NGY9K9opnRBQg:-L0JUsVkHxWctXq2_Z1v:b-896fix) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/c51376263402937f717199616bfeda4bb0040613/downloader/src/main/kotlin/VersionControlSystem.kt#L180)):*
<details><summary><i>Previously, mnonnenmacher (Martin Nonnenmacher) wrote…</i></summary><blockquote>

The empty string is used as a revision in VcsInfo below for WorkingDirectoryWithFileRevisions. I think we should consistently use the empty string then and not allow this to return null.
</blockquote></details>

Done.

---

*[downloader/src/main/kotlin/VersionControlSystem.kt, line 215 at r1](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JD0uOOW4PCGTnnUfp:-L0JUyNtxwKuMvgi7I9A:b-896fix) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/c51376263402937f717199616bfeda4bb0040613/downloader/src/main/kotlin/VersionControlSystem.kt#L215)):*
<details><summary><i>Previously, mnonnenmacher (Martin Nonnenmacher) wrote…</i></summary><blockquote>

> workingFir
workingDir
</blockquote></details>

Done.

---

*[downloader/src/main/kotlin/VersionControlSystem.kt, line 226 at r1](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JD6jUrJHpaFkBcTiq:-L0JV1Jywum1E9fKeFbu:b-896fix) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/c51376263402937f717199616bfeda4bb0040613/downloader/src/main/kotlin/VersionControlSystem.kt#L226)):*
<details><summary><i>Previously, mnonnenmacher (Martin Nonnenmacher) wrote…</i></summary><blockquote>

> workingFir
workingDir
</blockquote></details>

Done.

---

*[downloader/src/main/kotlin/vcs/Cvs.kt, line 36 at r2](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JHGnh3q1mtIHI4Ilh:-L0JVal9wQByCH-lvhC_:b-896fix) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/9fb84d1f26af5453201dff64569b33fd828349c6/downloader/src/main/kotlin/vcs/Cvs.kt#L36)):*
<details><summary><i>Previously, mnonnenmacher (Martin Nonnenmacher) wrote…</i></summary><blockquote>

Rename to cvsVersionRegex.
</blockquote></details>

Done.

---

*[downloader/src/main/kotlin/vcs/Cvs.kt, line 56 at r2](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JIJSyftyciqD-UpJF:-L0JX8JfOFpCyVpiO_Jr:b-896fix) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/9fb84d1f26af5453201dff64569b33fd828349c6/downloader/src/main/kotlin/vcs/Cvs.kt#L56)):*
<details><summary><i>Previously, mnonnenmacher (Martin Nonnenmacher) wrote…</i></summary><blockquote>

You can save one trim:
`commandOutput.map { it.trim() }.first { it.startsWith(revisionPrefix) }.removePrefix(revisionPrefix).trim()`
</blockquote></details>

Done.

---

*[downloader/src/main/kotlin/vcs/Cvs.kt, line 61 at r2](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JInw1JQsnmID4OjVJ:-L0JXJNJ9Vo-3jIXRREc:b-896fix) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/9fb84d1f26af5453201dff64569b33fd828349c6/downloader/src/main/kotlin/vcs/Cvs.kt#L61)):*
<details><summary><i>Previously, mnonnenmacher (Martin Nonnenmacher) wrote…</i></summary><blockquote>

Can we omit the return type here?
</blockquote></details>

Done.

---

*[downloader/src/main/kotlin/vcs/Cvs.kt, line 64 at r2](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JIxyxuksEp3jPdRTh:-L0JXbD69ZbhZN_1Oz18:b-896fix) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/9fb84d1f26af5453201dff64569b33fd828349c6/downloader/src/main/kotlin/vcs/Cvs.kt#L64)):*
<details><summary><i>Previously, mnonnenmacher (Martin Nonnenmacher) wrote…</i></summary><blockquote>

Can we omit the return type here?
</blockquote></details>

Done.

---

*[downloader/src/main/kotlin/vcs/Cvs.kt, line 78 at r2](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JLHM1Htgh0T2ePjCs:-L0JZce0JMeNFB1nXhVU:bf3szeu) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/9fb84d1f26af5453201dff64569b33fd828349c6/downloader/src/main/kotlin/vcs/Cvs.kt#L78)):*
<details><summary><i>Previously, mnonnenmacher (Martin Nonnenmacher) wrote…</i></summary><blockquote>

This provides much better performance for large files, because it does not read the full file to get only the first line:
`File(...).useLines { it.first() }`

You have this in several places, please replace all of them.
</blockquote></details>

Good to know, Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135)*
<!-- Sent from Reviewable.io -->

mnonnenmacher commented:
Reviewed 14 of 14 files at r4, 5 of 5 files at r5, 2 of 2 files at r6.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[downloader/src/main/kotlin/vcs/Cvs.kt, line 56 at r2](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JIJSyftyciqD-UpJF:-L0Jwy6Xo8C2VpUsl83D:b109wug) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/9fb84d1f26af5453201dff64569b33fd828349c6/downloader/src/main/kotlin/vcs/Cvs.kt#L56)):*
<details><summary><i>Previously, pnowakowski wrote…</i></summary><blockquote>

Done.
</blockquote></details>

Have you? Looks like it's still the same.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135)*
<!-- Sent from Reviewable.io -->

pnowakowski commented:
Review status: 10 of 13 files reviewed at latest revision, 1 unresolved discussion.

---

*[downloader/src/main/kotlin/vcs/Cvs.kt, line 56 at r2](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L0JIJSyftyciqD-UpJF:-L0JxoRRL767zAduutSF:b-ybagfv) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/9fb84d1f26af5453201dff64569b33fd828349c6/downloader/src/main/kotlin/vcs/Cvs.kt#L56)):*
<details><summary><i>Previously, mnonnenmacher (Martin Nonnenmacher) wrote…</i></summary><blockquote>

Have you? Looks like it's still the same.
</blockquote></details>

I think there was one more `trim()` before, however not sure anymore... Anyway changed implementation to reg expression to avoid those trims. 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135)*
<!-- Sent from Reviewable.io -->

mnonnenmacher commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 3 of 3 files at r7, 2 of 2 files at r8.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-:-L0KXyJWfZ1bUcwT5_qR:bnfp4nl)*
<!-- Sent from Reviewable.io -->

sschuberth commented on .travis.yml:
```diff
@@ -17,6 +17,9 @@ cache:
 
 install:
   - npm install -g npm@5.5.1 yarn@1.3.2
+  - if [ "$TRAVIS_OS_NAME" = "linux" ]; then
```
So far we only build on Linux with Travis, so strictly speaking that's not necessary.

sschuberth commented on .travis.yml:
```diff
@@ -17,6 +17,9 @@ cache:
 install:
   - npm install -g npm@5.5.1 yarn@1.3.2
+  - if [ "$TRAVIS_OS_NAME" = "linux" ]; then
+      sudo apt-get install --yes cvs libtool;
```
How about using the more modern `apt` instead of `apt-get`? And the more common `-y` instead of `--yes`? And why is `libtool` explicitly required here?

sschuberth commented on appveyor.yml:
```diff
@@ -17,7 +17,9 @@ install:
   - npm install -g npm@5.5.1 yarn@1.3.2
   - pip install virtualenv==15.1.0
   - cinst sbt --version 1.0.2 -y
+  - C:\msys64\usr\bin\bash -lc "pacman --noconfirm -Sy cvs"
```
Just FYI: I just double-checked that there's indeed [no Chocolatey package for CVS](https://chocolatey.org/packages?q=cvs) that we could use.

sschuberth commented on appveyor.yml:
```diff
@@ -17,7 +17,9 @@ install:
   - cinst sbt --version 1.0.2 -y
+  - C:\msys64\usr\bin\bash -lc "pacman --noconfirm -Sy cvs"
   - refreshenv
+  - set PATH=C:\msys64\usr\bin;%PATH%
```
Hmm, putting the whole of `C:\msys64\usr\bin` into `PATH` might lead to clashes, like with the `find` tool. However, I don't have a good idea how to address this right now.

mnonnenmacher replied:
Would it be better to change the order to `%PATH%;C:\msys64\usr\bin`?

sschuberth replied:
Yeah, I guess that's a good compromise.

pnowakowski commented:
Review status: 6 of 14 files reviewed at latest revision, 4 unresolved discussions.

---

*[.travis.yml, line 20 at r8](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L09Ycnc4AwxY5AWjH1H-r8-20:-L0NwYfQ7JDFZn7UlTAK:b-896fix) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/cf771c34c3f877187a5b6ef45fe815d3cd0e9682/.travis.yml#L20)):*
<details><summary><i>Previously, sschuberth (Sebastian Schuberth) wrote…</i></summary><blockquote>

So far we only build on Linux with Travis, so strictly speaking that's not necessary.
</blockquote></details>

Done.

---

*[.travis.yml, line 21 at r8](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L09Ycnc4AwxY5AWjH1H-r8-21:-L0O1e0Nrr_ieFMFJR2L:b-g6r7lo) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/cf771c34c3f877187a5b6ef45fe815d3cd0e9682/.travis.yml#L21)):*
<details><summary><i>Previously, sschuberth (Sebastian Schuberth) wrote…</i></summary><blockquote>

How about using the more modern `apt` instead of `apt-get`? And the more common `-y` instead of `--yes`? And why is `libtool` explicitly required here?
</blockquote></details>

Changed to `apt-get` to `apt` and `--yes` yo `-y`.
I've read somewhere _libtool_ is required by CVS, but looks like it's already there, so I removed it from the command.

---

*[appveyor.yml, line 22 at r8](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135#-L09Ycnc4AwxY5AWjH1I-r8-22:-L0Nx4dF5sOteG6EF-aK:b-896fix) ([raw file](https://github.com/heremaps/oss-review-toolkit/blob/cf771c34c3f877187a5b6ef45fe815d3cd0e9682/appveyor.yml#L22)):*
<details><summary><i>Previously, sschuberth (Sebastian Schuberth) wrote…</i></summary><blockquote>

Yeah, I guess that's a good compromise.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135)*
<!-- Sent from Reviewable.io -->

mnonnenmacher commented:
Reviewed 8 of 8 files at r9, 5 of 5 files at r10.
Review status: 12 of 14 files reviewed at latest revision, 4 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135)*
<!-- Sent from Reviewable.io -->

mnonnenmacher commented:
Reviewed 2 of 2 files at r11.
Review status: all files reviewed at latest revision, 4 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/heremaps/oss-review-toolkit/135)*
<!-- Sent from Reviewable.io -->

sschuberth commented:
Superseded by https://github.com/heremaps/oss-review-toolkit/pull/291.
